packages/core/src/routes/log.test.ts

```diff
@@ -42,8 +42,18 @@ describe('logRoutes', () => {
       await logRequest.get(
         `/logs?userId=${userId}&applicationId=${applicationId}&logKey=${logKey}&page=${page}&page_size=${pageSize}`
       );
-      expect(countLogs).toHaveBeenCalledWith({ userId, applicationId, logKey });
-      expect(findLogs).toHaveBeenCalledWith(5, 0, { userId, applicationId, logKey });
+      expect(countLogs).toHaveBeenCalledWith({
+        userId,
+        applicationId,
+        logKey,
+        includeWebhookLogs: false,
```

Comment:
I think we can tell if the API is requesting webhook logs by just checking the `logKey` alone. Therefore it seems we don't necessarily have to add the new `includeWebhookLogs` param, no?